app/controllers/servers_controller.rb

```diff
@@ -7,6 +7,7 @@ class ServersController < ApplicationController
 
   def index
     @servers = Server.includes(:frame, :room, :islet, bay: :frames, modele: :category).order(:name)
+    @servers = @servers.where(name: params[:name]) if params[:name].present?
```

Comment:
```suggestion
    @servers = @servers.where(Server.arel_table[:name].matches("%#{params[:name]}%")) if params[:name].present?
```

I suggest the usage of `LIKE` SQL, to let user search with partial name.

Reply:
Indeed, thanks